packages/ckeditor5-ui/src/menubar/utils.ts

```diff
@@ -824,7 +825,8 @@ export const DefaultMenuBarItems: MenuBarConfigObject[ 'items' ] = [
 				items: [
 					'menuBar:link',
 					'menuBar:comment',
-					'menuBar:insertMergeField'
+					'menuBar:insertMergeField',
+					'menuBar:bookmark'
```

Comment:
Logically, this makes sense to be closer to link, as it has a similarity to it. 

Please fill in the standard GSheet with placements https://docs.google.com/spreadsheets/d/1bb3FzKZlxpT7elO_AFm_4O2eTYsmktNsWqwVVd1p5AM/edit?pli=1&gid=0#gid=0